bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/discovery/ThingDiscoveryService.java

```diff
@@ -109,9 +113,17 @@ private void addLight(String lightID, LightMessage light) {
             return;
         }
 
-        if (light.uniqueid.isEmpty()) {
-            logger.warn("No unique id reported for light {} ({})", light.modelid, light.name);
-            return;
+        Map<String, Object> properties = new HashMap<>();
+        properties.put("id", lightType);
+        properties.put(UNIQUE_ID, light.uniqueid);
+        properties.put(Thing.PROPERTY_FIRMWARE_VERSION, light.swversion);
+        properties.put(Thing.PROPERTY_FIRMWARE_VERSION, light.swversion);
+        properties.put(Thing.PROPERTY_VENDOR, light.manufacturername);
+        properties.put(Thing.PROPERTY_MODEL_ID, light.modelid);
+
+        if (light.ctmax != null && light.ctmin != null) {
+            properties.put(PROPERTY_CT_MAX, Integer.toString(light.ctmax));
```

Comment:
define `private static final int CT_MAX = 500;` in this class

```suggestion
            int ctmax = light.ctmax > 500 ? 500 : light.ctmax;
            properties.put(PROPERTY_CT_MAX, Integer.toString(ctmax));
```

and the same for min